core/container/kubernetescontroller/kubernetescontroller.go

```diff
@@ -226,6 +236,73 @@ func (api *KubernetesAPI) createChaincodePodDeployment(ccid ccintf.CCID, args []
 	return api.client.Core().Pods(api.Namespace).Create(pod)
 }
 
+func getResourceQuantity(key string) (*resource.Quantity, error) {
+	q := viper.GetString(key)
+	if q == "" {
+		// Not specified in config.
+		return nil, nil
+	}
+
+	v, err := resource.ParseQuantity(q)
+	if err != nil {
+		return nil, err
+	}
+
+	return &v, nil
+}
+
+func getResourceRequest() (apiv1.ResourceRequirements, error) {
```

Comment:
Does this already have the necessary RBAC?

Reply:
The peer pod already has the authorization to create new pods. Is there a different access control to adjust resource requests during pod creation that is different than "create"?

Reply:
Maybe @iramiller can chime in as the resident helm expert as to the setup of the access controls.

Reply:
```
Ira [9:24 AM]
I see your comment  on resource requests …

-------------------------
kind: Role
apiVersion: rbac.authorization.k8s.io/v1
metadata:
  namespace: {{ .Release.Namespace }}
  name: {{ .Release.Namespace }}-peer-chaincode-manager
rules:
- apiGroups: ["", "extensions", "apps"]
  resources: ["deployments", "replicasets", "pods", "configmaps"]
  verbs: ["get", "list", "watch", "create", "update", "patch", "delete"]
-------------------------
```